packages/xo-server-perf-alert/src/index.js

```diff
@@ -524,11 +524,39 @@ ${monitorBodies.join('\n')}`
                   })
                 }
 
-                result.listItem = `  * ${result.objectLink}: ${
-                  result.value === undefined
-                    ? "**Can't read performance counters**"
-                    : result.value.toFixed(1) + typeFunction.unit
-                }\n`
+                const getListItem = () => {
+                  if (result.value === undefined) {
+                    return "**Can't read performance counters**"
+                  }
+
+                  if (lcObjectType === 'vm' && definition.variableName === 'memoryUsage') {
+                    const checkManagementAgent = (vm, guestMetrics) => {
+                      if ((vm.power_state !== 'Running' && vm.power_state !== 'Paused') || guestMetrics === undefined) {
+                        return
+                      }
+
+                      const { major, minor } = guestMetrics.PV_drivers_version
+                      const hasPvVersion = major !== undefined && minor !== undefined
+                      return hasPvVersion || guestMetrics.other['feature-static-ip-setting'] === '1'
+                    }
```

Comment:
It doesn't look like this function needs to be declared in `getListItem`'s context, you can extract it outside.